CMakeLists.txt

```diff
@@ -26,8 +26,13 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-cmake_minimum_required(VERSION 3.13.1)
-project(openthread)
+cmake_policy(SET CMP0048 NEW)
+cmake_minimum_required(VERSION 3.11.4)
```

Comment:
How is the minimum version determined? Are there dependent features on version 3.11.4?

Reply:
It is the cmake in openwrt 18.06, the version we support in ot-br-posix.